### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by sugargoat**
### Motivation

We moved quickly on the last PR to meet the timing needs of a partner. This PR cleans up some issues in the repo. 

### In this PR
* Remove the integration test in favor of planning a longer term testing strategy. Even though some tests are better than no tests, bash scripts are not the most robust testing framework.
* Fix get_block_info endpoint 
* Update the release script to replace the version for the db directories
* Remove the testnet-client release script, as that should live in mobilecoin, despite it being convenient to run both release scripts from here.

### CODE REVIEW

**Review by tsegaran — APPROVED** [review]

**Review by eranrund — APPROVED** [review]
This is good but the issue of https://github.com/mobilecoinofficial/mobilecoind-mirror/blob/master/src/private/main.rs#L345 not supporting the two new requests remains.
Let me know if you'd like me to address it.
Thanks for cleaning it up!

**Review by jcape — CHANGES_REQUESTED** [review]
`.DS_Store` file needs to not be in this repo ;-)

File: create-release/create-release.sh [hunk 1]
```diff
@@ -18,6 +19,9 @@ cargo build --manifest-path $PROJECT_ROOT/Cargo.toml --release
 # Create release dir
 mkdir $RELEASE_NAME
 cp -R $SCRIPT_DIR/package/* $RELEASE_NAME
+sed -i 's/VERSION/'${VERSION}'/g' $RELEASE_NAME/mobilecoind-mirror-private.sh
+sed -i 's/VERSION/'${VERSION}'/g' $RELEASE_NAME/mobilecoind-mirror-private-tls.sh
+sed -i 's/VERSION/'${VERSION}'/g' $RELEASE_NAME/mobilecoind-mirror-private-tls-encrypted.sh
```
Comment (jcape) [inline create-release/create-release.sh:24]: Why isn't this just in the file?

File: create-release/package/mobilecoind-mirror-private-tls-encrypted.sh [hunk 2]
```diff
@@ -15,12 +15,12 @@ if [ ! -f "mirror-private.pem" ]; then
 fi
 
 
-./bin/mobilecoind --ledger-db /tmp/mobilecoin/0.6.0/ledger \
+./bin/mobilecoind --ledger-db /tmp/mobilecoin/VERSION/ledger \
```
Comment (jcape) [inline create-release/package/mobilecoind-mirror-private-tls-encrypted.sh:18]: ...i.e. why not just make this be `${VERSION}` directly.

**Review by jcape — APPROVED** [review]
Assuming ds store is removed

**Comment by eranrund** [comment]
I'm picking this up, will re-submit from my own fork in a bit.

**Pull request closed without merging**